clients/filesystem-fuse/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+[package]
+name = "filesystem-fuse"
+version = "0.8.0-incubating-SNAPSHOT"
+edition = "2021"
+rust-version = "1.75"
+
+[[bin]]
+name = "gvfs-fuse"
+path = "src/main.rs"
+
+[dependencies]
+fuse3 = { version = "0.8.1", "features" = ["tokio-runtime", "unprivileged"] }
```

Comment:
There are many fuse implement in RUST, why you choose fuse3?

Reply:
seems fuser is more popular

Reply:
Yes, fuser is more popular, but FUSE3 is used by OpenDAL. We need to decide which one to use soon